azure/functions/timer.py

```diff
@@ -32,5 +39,8 @@ def decode(cls, data: meta.Datum, *, trigger_metadata) -> typing.Any:
             raise NotImplementedError
 
         info = json.loads(data.value)
+
         return TimerRequest(
-            past_due=info.get('IsPastDue', False))
+            past_due=info.get('IsPastDue', False),
+            schedulestatus=info["ScheduleStatus"],
```

Comment:
We should probably keep this the same as how we retrieve pastdue with a default value.  e.g info.get("ScheduleStatus" , {})

Reply:
Yes - what if info doesn't have ScheduleStatus - it would throw key error. - we should take care of this.

Reply:
I have updated this change as per recommendation.